### DISCUSSION TITLE
About the course.

### DISCUSSION THREAD

**Discussion by AliHaider20** [microsoft/ML-For-Beginners#120] [Q&A] [answered]
Hi all, I just want to know about the course. Can anybody let me know the timeline and where from where to take pre lecture quiz?

**Comment by Rishit-dagli** [comment, marked as answer]
Hi @AliHaider20 ,
As I understand you should be managing your time as mentioned by @jlooper in https://github.com/microsoft/ML-For-Beginners/discussions/75#discussioncomment-957060 . About the quizzes, all of them are present on https://jolly-sea-0a877260f.azurestaticapps.net/ , you would also find the pre-lecture quizzes for each lesson linked in their README.
